### DISCUSSION TITLE
Prevent eslint from linting workspace packages

### DISCUSSION THREAD

**Discussion by purecopy** [gatsbyjs/gatsby#28210] [Help]
Hi, I am currently facing some weird troubles with integrating Gatsby in an existing project using Yarn workspaces and shared components which are already transpiled to esm/cjs by an separate rollup task. Starting from the gatsby-starter-default and importing my shared library, Gatsby unnecessarily starts linting the transpiled code again with lots of errors.

What am I missing out?

**Comment by ascorbic** [comment]
Have you tried adding an [.eslintignore](https://eslint.org/docs/user-guide/configuring#ignoring-files-and-directories)?
